### PR TITLE
Add index, refresh tokens and examples

### PR DESCRIPTION
**Pull request by OrthoLoess**
Added content to index page.
Made page about refresh tokens and added info on how to use access tokens
Added example HTTP requests.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1 +1,12 @@
-# Test
+Welcome to the 3rd Party Developer Documentation for EVE Online.
```
Comment (Regner) [inline docs/index.md]: All references to 3rd party should be typed out fully as "third-party developer(s)"

File: docs/index.md [hunk 2]
```diff
@@ -1 +1,12 @@
...
+
+This is a new, community supported repository for instructions and guidance that is useful for those making 3rd party tools for EVE.
+
+It is intended to always be a living document, and the plan is for it to be frequently updated to reflect changes in the APIs and/or services. At present it is very much a work in progress and much of the content is yet to be written. The main focus so far has been on the SSO and CREST, as these are new services, which have not yet had extensive third party documentation efforts, and get asked about frequently on the [dev forums](https://forums.eveonline.com/default.aspx?g=topics&f=263).
```
Comment (Regner) [inline docs/index.md]: The link for the tech lab should be called "technology lab"

File: docs/crest/authentication.md [hunk 3]
```diff
@@ -15,3 +15,18 @@ The first scope released. It allows access to live market orders data for any re
 
 ### characterStatisticsRead - SISI
 Currently only available on SISI. Allows access to historical character statistics, broken down by year. This was announced in the EVE Keynote at FanFest 2015.
+
+# Using your access token
+Once you have a valid access token from the SSO, you can make call to authenticated crest by adding an Authorization header and using the auth'd crest endpoint.
```
Comment (Regner) [inline docs/crest/authentication.md]: CREST should always be referenced as CREST

File: docs/index.md [hunk 4]
```diff
@@ -1 +1,12 @@
...
+
+# Contributing
+There is a lot of work to be done on this site and any contributions are welcome. The docs are hosted on a [github repo](https://github.com/ccpgames/eveonline-third-party-documentation), and anyone can fork it, commit changes and submit a pull request to get your changes onto the site.
```
Comment (Regner) [inline docs/index.md]: Would rather this said repository than repo.

Comment (Regner) [inline docs/index.md]: To many and's in the second sentence. What about ", where anyone can fork it...."

Comment (Regner) [inline docs/index.md]: GitHub is a proper name as well should should be spelled out as GitHub with the capitalization.

File: docs/index.md [hunk 5]
```diff
@@ -1 +1,12 @@
...
+
+The source files are written in markdown, which is very easy to get started using. There is a fairly good introduction to markdown available from [github](https://guides.github.com/features/mastering-markdown/).
+
+There is a contributors file, as well as more info on contributing to the documentation on the repo.
```
Comment (Regner) [inline docs/index.md]: GitHub

File: docs/index.md [hunk 6]
```diff
@@ -1 +1,12 @@
...
+
+This is a new, community supported repository for instructions and guidance that is useful for those making 3rd party tools for EVE.
+
+It is intended to always be a living document, and the plan is for it to be frequently updated to reflect changes in the APIs and/or services. At present it is very much a work in progress and much of the content is yet to be written. The main focus so far has been on the SSO and CREST, as these are new services, which have not yet had extensive third party documentation efforts, and get asked about frequently on the [dev forums](https://forums.eveonline.com/default.aspx?g=topics&f=263).
+
+# Contributing
+There is a lot of work to be done on this site and any contributions are welcome. The docs are hosted on a [github repo](https://github.com/ccpgames/eveonline-third-party-documentation), and anyone can fork it, commit changes and submit a pull request to get your changes onto the site.
+
+The source files are written in markdown, which is very easy to get started using. There is a fairly good introduction to markdown available from [github](https://guides.github.com/features/mastering-markdown/).
```
Comment (Regner) [inline docs/index.md]: GitHub

File: docs/index.md (hunk 5 above)
Comment (Regner) [inline docs/index.md]: repo should be repository.

File: docs/index.md [hunk 7]
```diff
@@ -1 +1,12 @@
-# Test
+Welcome to the 3rd Party Developer Documentation for EVE Online.
+
+This is a new, community supported repository for instructions and guidance that is useful for those making 3rd party tools for EVE.
```
Comment (Regner) [inline docs/index.md]: third-party

Comment (Regner) [inline docs/index.md]: Either there needs to be a second comma after community supported, or the first comma after new should be removed. I would personally lean towards removing the first comma.

File: docs/sso/refreshtokens.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+If any valid scope was requested in the initial redirect to the SSO, a refresh token will be returned by the token endpoint, along with the access token. While the access token will expire after a set interval - currently 20 minutes, the refresh token can be stored and used indefinitely. (Users can revoke access for individual apps on the support site)
+
+To get a new access token you make a POST request to "https://login.eveonline.com/oauth/token" with the following params:
+
+    grant_type=refresh_token&refresh_token=USERS_REFRESH_TOKEN
```
Comment (Regner) [inline docs/sso/refreshtokens.md]: Shouldn't this be in the headers? I always thought it was, never actually done it manually myself.

File: docs/crest/authentication.md (hunk 3 above)
Comment (OrthoLoess) [inline docs/crest/authentication.md]: Dang, I was doing so well at bothering to capitalise it!

File: docs/sso/refreshtokens.md (hunk 8 above)
Comment (OrthoLoess) [inline docs/sso/refreshtokens.md]: I think I tried doing that and it failed. I'm not certain though, and checking will have to wat until tomorrow when I'm back at my dev machine.

**Comment by OrthoLoess** [comment]
I think I've fixed everything except the last one. I clearly got a little bit lazy and/or tired with regards to style on this one compared to the last pull request :P

File: docs/sso/refreshtokens.md (hunk 8 above)
Comment (islador) [inline docs/sso/refreshtokens.md]: It works as both a header or a param as per my testing. 

HOWEVER
as per the documentation on developers.eveonline.com; https://developers.eveonline.com/resource/single-sign-on

It's a param.

Comment (OrthoLoess) [inline docs/sso/refreshtokens.md]: Happy to write it as either. If you want it done as headers, the other page (copied from dev site) will presumably want changing to that as well. (and maybe even the dev site itself!)

Comment (Regner) [inline docs/sso/refreshtokens.md]: Apparently param is what we expect so hey thats what we shall document. :)

File: docs/crest/authentication.md [hunk 9]
```diff
@@ -15,3 +15,18 @@ The first scope released. It allows access to live market orders data for any re
 
 ### characterStatisticsRead - SISI
 Currently only available on SISI. Allows access to historical character statistics, broken down by year. This was announced in the EVE Keynote at FanFest 2015.
+
+# Using your access token
+Once you have a valid access token from the SSO, you can make call to authenticated CREST by adding an Authorization header and using the auth'd crest endpoint.
```
Comment (Regner) [inline docs/crest/authentication.md]: calls vs call

File: docs/index.md [hunk 10]
```diff
@@ -1 +1,12 @@
-# Test
+Welcome to the Third Party Developer(s) Documentation for EVE Online.
+
+This is a new community supported repository for instructions and guidance that is useful for those making third party tools for EVE.
```
Comment (Regner) [inline docs/index.md]: third-party please :)

File: docs/index.md [hunk 11]
```diff
@@ -1 +1,12 @@
-# Test
+Welcome to the Third Party Developer(s) Documentation for EVE Online.
```
Comment (Regner) [inline docs/index.md]: Third-Party (this is what I was told is appropriate)

**Pull request merged**